extensions/amp-form/0.1/amp-form.js

```diff
@@ -192,7 +197,10 @@ function installSubmissionHandlers(win) {
 function installAmpForm(win) {
   return getService(win, 'amp-form', () => {
     if (isExperimentOn(win, TAG)) {
-      installStyles(win.document, CSS, () => installSubmissionHandlers(win));
+      installStyles(win.document, CSS, () => {
+        installSubmissionHandlers(win);
+        actionServiceFor(win).addEvent('submit');
```

Comment:
I'd say, everything is driven from the meaning of this event. If, as I suspect, you'd like to get a "submit-success" event - then this implementation is not correct. A better bet would be to just trigger the event directly using `ActionService.trigger` method.